questionnaire/form-helper.js

```diff
@@ -47,11 +47,13 @@ function renderSection({
     showBackLink = true,
     csrfToken,
     cspNonce,
+    loggedInUser,
     showSignInLink = shouldShowSignInLink(sectionId, uiSchema)
 }) {
     const showButton = !isFinal;
     const buttonTitle = getButtonText(sectionId);
     const hasErrors = transformation.hasErrors === true;
+    const showLink = showSignInLink && !loggedInUser;
```

Comment:
is `showLink` too "general". Could we add logic to the showSignInLink function to check is a user is authenticated? Could slim things down a bit